### ISSUE TITLE
Help wanted with adding more to w:rPr (Styles for: Font, Footer, TOC, Field, Numbering, etc)

### ISSUE THREAD

**Issue by FBnil** [PHPOffice/PHPWord#1164]
This is a list of all it can expect:  http://www.datypic.com/sc/ooxml/e-w_rPr-1.html
This is a list of what it read's in: [./src/PhpWord/Reader/Word2007/AbstractPart.php](https://github.com/PHPOffice/PHPWord/blob/develop/src/PhpWord/Reader/Word2007/AbstractPart.php#L343)
This is what it does NOT read:

    w:outline [0..1]    Display Character Outline
    w:shadow [0..1]    Shadow
    w:emboss [0..1]    Embossing
    w:imprint [0..1]    Imprinting
    w:noProof [0..1]    Do Not Check Spelling or Grammar
    w:snapToGrid [0..1]    Use Document Grid Settings For Inter-Character Spacing
    w:vanish [0..1]    Hidden Text
    w:webHidden [0..1]    Web Hidden Text
    w:spacing [0..1]    Character Spacing Adjustment
    w:w [0..1]    Expanded/Compressed Text
    w:kern [0..1]    Font Kerning
    w:position [0..1]    Vertically Raised or Lowered Text
    w:effect [0..1]    Animated Text Effect
    w:shd [0..1]    Run Shading
    w:fitText [0..1]    Manual Run Width
    w:cs [0..1]    Use Complex Script Formatting on Run
    w:em [0..1]    Emphasis Mark
    w:lang [0..1]    Languages for Run Content
    w:eastAsianLayout [0..1]    East Asian Typography Settings
    w:specVanish [0..1]    Paragraph Mark Is Always Hidden
    w:oMath [0..1]    Office Open XML Math

Are there any tags that are important to add? Is it necessary to add all of it? Is PHPWord a little lacking so that the paid versions shine a little better?

./src/PhpWord/Style/Font.php looks a bit better, supporting things like kerning and shading, but not all of the possible OOXML properties.

And if you read the other issues, you know that things like ListItems do not support any Styling at all (beyond the basics). So my idea is start there, adding, just like TOC has the option to set a font name OR a font object (with all the properties and let the XMLwriters take care of the handling).
But everything touches everything, so I wanted to know your opinions.

**Comment by troosan** [comment]
Indeed, a lot of those are still missing. I do not have the time to systematically implement all of those. Right now new features are being added on request.

**Comment by christian-kolb** [comment]
We would love to have the numbers of an ordered list being affected by a bold styling. Is this also part of this or is this another topic?

**Comment by Progi1984** [comment]
@christian-kolb Have you got a sample code, please, for implementation ?

**Comment by christian-kolb** [comment]
@Progi1984 Yes, I realized that it makes more sense to have a dedicated issue for it, so I created one with example code and expected output here: https://github.com/PHPOffice/PHPWord/issues/2723
